lib/collections/schemas/helpers.js

```diff
@@ -55,3 +56,25 @@ export function schemaIdAutoValue() {
   }
   return this.unset();
 }
+
+/**
+ * shopDefaulCountry
+ * @summary used for schema injection autoValue
+ * @example autoValue: shopDefaultCountry
+ * @return {String} returns country value from default shop
+ */
+export function shopDefaultCountry() {
+  if (this.isSet && Meteor.isServer) {
+    return this.value;
+  } else if (Meteor.isServer && !this.isUpdate || Meteor.isClient && this.isInsert) {
+    const shop = Shops.findOne({
+      _id: Reaction.getShopId()
+    });
+    if (shop && shop.addressBook && shop.addressBook[0]) {
```

Comment:
I don't love that we're assuming that the 0th address is the correct address here, but I know that's a pattern that's older than this PR.

Is there an indicator that a shop is using a specific address as their "shop address" similar to how customers can have a default address? Is it the default flag? Or any other way to identify the Shop's main address?

Reply:
I don't believe there is any way to determine the `default` address, and calling on the `[0]` is the way to do it.

Reply:
Actually, there's a field on the schema that determines this.
```
  isBillingDefault: {
    label: "Make this your default billing address?",
    type: Boolean
  },
  isShippingDefault: {
    label: "Make this your default shipping address?",
    type: Boolean
  },
```

Reply:
Do we not use that for Shops?

Reply:
Is see that it's currently set to True on one created shop. Even if we don't set it for now this feels like a more "future-proof" way of doing it.

Reply:
^^ yeah... dunno why I didn't connect that, will update to use... one of the two. Which should be used for this? Shipping?

This is for the product origin.

Reply:
I think Shipping makes the most sense. That where you would have things shipped to is there you want to ship from. At least in most cases and as accurate as being the right country.

Reply:
So we do set it explicitly in the dashboard so I think that's the way to go. Maybe we could add a note there that says setting that will set the default origin for Products? It's probably an autoform so not sure how hard that would be.

![reaction](https://user-images.githubusercontent.com/72819/30458735-773da4bc-99df-11e7-8362-3a095f301011.png)